lib/commands/list-private.js

```diff
@@ -17,7 +17,8 @@ function formatChannel (chan) {
 
 function displayGroupList (filter, groups) {
   const str = groups.reduce((acc, group) => (
-    `${acc}  ~ ${formatChannel(group)}\n`
+    // Subtracting 1 from `group.members.length` to exclude Wheelie.
+    `${acc}  ~ ${formatChannel(group)} (${group.members.length - 1})\n`
```

Comment:
oh. Yeah idk. This list can't be displayed at all without wheelie there, so :)

I'd need to track (and/or request) bot names across different slacks that might have different names. And then deal with potential weirdness in the match.